.github/cicd/functions.sh

```diff
@@ -0,0 +1,174 @@
+export BRANCH_LOWER=`echo "${BRANCH_NAME}" | awk '{print tolower($0)}'`
+function variablePopulation() {
+    if [ "${BRANCH_LOWER}" == "develop" ] || [ "${BRANCH_LOWER}" == "master" ];
+    then
+        export SUFFIX=""
+        export CHANGE_BRANCH="$BRANCH_NAME"
+    else
+        export SUFFIX="-${BRANCH_LOWER}";
+    fi
+    if [ "${APP_NAME}" == "test" ] || [ "${APP_NAME}" == "prod" ];
+    then
+        export DOTNET_PHASE="Release"
+    else
+        export DOTNET_PHASE="Development"
+    fi
+}
+
+variablePopulation
```

Comment:
comment?

Reply:
Sure, I'll add one.  Just executes the function I just built.